contracts/YearnHarvest.sol

```diff
@@ -361,6 +382,8 @@ contract YearnHarvest {
         }
     }
 
+    // REVIEW: Sometimes we add a sweepETH as well.
```

Comment:
sweep() takes care of tokens and native crypto. If _token = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE, the function sweeps FTM.  ETH/MATIC/FTM are defined  as 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE for Gelato's purposes